.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: php
 
+php: [5.3.3, 5.3, 5.4, 5.5, 5.6, 7.0, hhvm]
```

Comment:
Is this change really needed? Imho it makes the diff bigger than necessary and makes it harder in the future to track when a really necessary change was done.